From 2d8525f0f6c3f4ee528b689bd19b3a562225764c Mon Sep 17 00:00:00 2001
Message-ID: <2d8525f0f6c3f4ee528b689bd19b3a562225764c.1705854493.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705854493.git.sharlatanus@gmail.com>
References: <cover.1705854493.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 14 Jan 2024 21:15:43 +0000
Subject: [PATCH 08/19] gnu: python-stcal: Update to 1.5.2.

* gnu/packages/astronomy.scm (python-stcal): Update to 1.5.2.
[propagated-inputs]: Add python-asdf-3.0 and python-gwcs.
[native-inputs]: Remove python-cython; add python-cython-3 and python-semantic-version.

Change-Id: I8b6a90a3c0b75ee43de4c7069950a4f9578ea825
---
 gnu/packages/astronomy.scm | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 4c867aa66a..ce84b50e54 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2775,13 +2775,13 @@ (define-public python-stsci-stimage
 (define-public python-stcal
   (package
     (name "python-stcal")
-    (version "1.4.4")
+    (version "1.5.2")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "stcal" version))
        (sha256
-        (base32 "031ldihdmsdrwz4wl49bfk2bxmzbp1i3kidrw46xz936765zmnc0"))))
+        (base32 "0r1a9f476f8ivhrwbcny3rnj9s4plaps6p36c22w73g546d476b7"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -2796,16 +2796,19 @@ (define-public python-stcal
               (invoke "python" "setup.py" "build_ext" "--inplace"))))))
     (propagated-inputs
      (list opencv ;Provides OpenCV-Python
+           python-asdf-3.0
            python-astropy
+           python-gwcs
            python-numpy
            python-scipy))
     (native-inputs
-     (list python-cython
+     (list python-cython-3
            python-psutil
            python-pytest
            python-pytest-cov
            python-pytest-doctestplus
            python-pytest-openfiles
+           python-semantic-version
            python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/stcal")
     (synopsis "STScI tools and algorithms used in calibration pipelines")
-- 
2.41.0

